ThingIFSDK/ThingIFSDK/ThingIFAPI.swift

```diff
@@ -780,6 +780,21 @@ open class ThingIFAPI: Equatable, NSCoding {
 
     /** Aggregate history states
 
+     `AggregatedValueType` represents type of calcuated value with
+     `Aggregation.FunctionType`.
+
+     - If the function is `Aggregation.FunctionType.max`,
+       `Aggregation.FunctionType.min` or
+       `Aggregation.FunctionType.sum`, tye type may be same as type of
```

Comment:
I think `may be` is better because it depends on developers.

Even If a developer uses `Aggregation.FunctionType.min` to a `Aggregation.FieldType.decimal` field, the developer can get the value as `Int`.

If we use `must be`, it seems that we forbid getting value as `Int` from a field whose type is `Aggregation.FieldType.decimal`. I think that is overstatement.

What do you think?